joomla-projects/gsoc16_improved-multi-lingual
# Adding context title to the list view

Pull Request for Issue #94 -> Add "Component: Item Type" title to views
### Testing Instructions

Go to the list view and change item type, it should change in title, was also tested in a RTL language.
